### PR TITLE
Inventory

### PR DESCRIPTION
**Pull request by Petrica81**
Implemented an Inventory scriptable object and connected it with coins and potions from HUD.
Implemented a Pause canvas which contains buttons and the following panels: pause, inventory, options.

### CODE REVIEW

File: Assets/Scripts/ScriptableObjects/Inventory.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{
+    [Range(0,999)]
+    public int coins;
+    [Range(0,9)]
+    public int potions;
+
+    //Limit the number of coins
+    public void CheckCoins()
+    {
+        coins = Mathf.Clamp(coins, 0, 999);
```
Comment (widdrr) [inline Assets/Scripts/ScriptableObjects/Inventory.cs:16]: Is this required even after setting the Range attribute?

**Review by widdrr — APPROVED** [review]
seems good

File: Assets/Scripts/ScriptableObjects/Inventory.cs (hunk 1 above)
Comment (Petrica81) [reply Assets/Scripts/ScriptableObjects/Inventory.cs:16]: Range attribute doesn't allow you  to set a number out of range with unity inspector, while a code sequence can.

**Review by RobertKeanu — APPROVED** [review]
lgtm

**Pull request merged**